src/bundle/Resources/views/themes/admin/ui/component/dropdown.html.twig

```diff
@@ -20,7 +20,7 @@
     <div class="ibexa-dropdown__wrapper">
         <ul
             class="ibexa-dropdown__selection-info"
-            data-template="{{ include('@ibexadesign/ui/component/dropdown_selected_item.html.twig', {
+            data-selected-item-template="{{ include('@ibexadesign/ui/component/dropdown_selected_item.html.twig', {
```

Comment:
I think it's better to leave it as template, it's genera property name in whole system for this kind of things